contractcourt/chain_arbitrator_test.go

```diff
@@ -1 +1,277 @@
 package contractcourt
+
+import (
+	"bytes"
+	"io/ioutil"
+	"math/rand"
+	"net"
+	"os"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/channeldb"
+	"github.com/lightningnetwork/lnd/keychain"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/shachain"
+)
+
+var (
+	key = [chainhash.HashSize]byte{
+		0x81, 0xb6, 0x37, 0xd8, 0xfc, 0xd2, 0xc6, 0xda,
+		0x68, 0x59, 0xe6, 0x96, 0x31, 0x13, 0xa1, 0x17,
+		0xd, 0xe7, 0x93, 0xe4, 0xb7, 0x25, 0xb8, 0x4d,
+		0x1e, 0xb, 0x4c, 0xf9, 0x9e, 0xc5, 0x8c, 0xe9,
+	}
+
+	privKey, pubKey = btcec.PrivKeyFromBytes(btcec.S256(), key[:])
+	testTx          = &wire.MsgTx{
+		Version: 1,
```

Comment:
FWIW, the `codec.go` file in `channeldb` assumes that all transactions are version 2.

Reply:
Interesting, any reason we are still using version 1 some places? (e.g. https://github.com/lightningnetwork/lnd/blob/985902be2779a5dcc7ac1d5b3d7eb7929608816d/lnwallet/wallet.go#L688)

Reply:
Mostly just legacy code really. We should likely update the `wire` package to always use version 2.